pyproject.toml

```diff
@@ -211,6 +216,7 @@ DEP002 = [
     "mypy",
     "mypy-protobuf",
     "pre-commit",
+    "PyGithub",
```

Comment:
what is this used for?

Reply:
capa-rules script create-release uses this library: https://github.com/mandiant/capa-rules/blob/master/.github/scripts/create_releases.py

so addressing the Deptry complaint. Sorry it's not in its own PR.

Reply:
ha, I've added that... it's in a different repo so really required here?

Reply:
deptry was complaining for me 🤷‍♂️. i assume it doesn't know about the submodule boundary. do you want me to remove this?

Reply:
it's helpful during dev, so let's leave